composer/algorithms/ghost_batchnorm/README.md

```diff
@@ -0,0 +1,97 @@
+# 👻 Ghost BatchNorm
+
+[\[How to Use\]](#how-to-use) - [\[Suggested Hyperparameters\]](#suggested-hyperparameters) - [\[Technical Details\]](#technical-details) - [\[Attribution\]](#attribution)
+
+`Computer Vision`
+
+During training, BatchNorm normalizes a batch of inputs to have a mean of 0 and variance of 1.
+Ghost BatchNorm instead splits the batch into multiple "ghost" batches, each containing `ghost_batch_size` samples, and normalizes each one to have a mean of 0 and variance of 1.
+This causes training with a large batch size to behave more similarly to training with a small batch size and acts as a regularizer.
+
+| ![Ghost BatchNorm](ghost-batch-normalization.png) |
+|:--:
+|*A visualization of different normalization methods on the activation map in a neural network with multiple channels. M represents the batch dimension, C represents the channel dimension, and F represents the spatial dimensions (such as height and width). Ghost BatchNorm (upper right) is a modified version of BatchNorm that normalizes the mean and variance for disjoint sub-batches of the full batch. This image is Figure 1 in [Dimitriou & Arandjelovic, 2020](https://arxiv.org/abs/2007.08554).*|
+
+## How to Use
+
+### Functional Interface
+
+TODO(DAVIS): FIX
+
+```python
+def training_loop(model, train_loader):
+  opt = torch.optim.Adam(model.parameters())
+  loss_fn = F.cross_entropy
+  model.train()
+  
+  for epoch in range(num_epochs):
+      for X, y in train_loader:
+          y_hat = model(X)
+          loss = loss_fn(y_hat, y)
+          loss.backward()
+          opt.step()
+          opt.zero_grad()
```

Comment:
```suggestion
from composer import functional as cf

def training_loop(model, train_loader):
    opt = torch.optim.Adam(model.parameters())
    
    # only need to pass in opt if apply_blurpool is used after optimizer creation
    # otherwise only the model needs to be passed in
    cf.apply_ghost_batchnorm(model, ghost_batch_size=32, optimizers=opt)

    loss_fn = F.cross_entropy
    model.train()
  
    for epoch in range(num_epochs):
        for X, y in train_loader:
            y_hat = model(X)
            loss = loss_fn(y_hat, y)
            loss.backward()
            opt.step()
            opt.zero_grad()
```